### ISSUE TITLE
No premake script (premake4.lua) found

### ISSUE THREAD

**Issue by KingBoomie** [cgbystrom/twinkle#1]
Just running "premake4.exe vs2013" will give the error in the title. Moving the lua script to the same folder as premake will fix it.

**Comment by cgbystrom** [comment]
Yeah, you got to run ```bin\premake4 vs2013``` from the top directory.
